docs/backup-restore.md

```diff
@@ -0,0 +1,101 @@
+# Scalar DL Backup Creation and Restoration
+
+Since Scalar DL provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
+This guide shows you how to create and restore transactionally-consistent Scalar DL backups.
+
+## How to create transactionally consistent backup 
+
+Scalar DL service is integrated with the admin interface, which allows you to pause the Scalar DL services using the scalar-admin client tool.
+
+### Requirements
+
+* You must wait at least 10 sec after pausing to create a backup.
+* You must be able to identify a common restore point for the Ledger and Auditor If you use Auditor.
+
+### Pause
+
+The `PAUSE` command helps you to create a transactionally consistent backup for non-transactional databases. It helps to pause the Scalar DL service.
+
+```console
+docker run -it --rm ghcr.io/scalar-labs/scalar-admin:<version> -c PAUSE -s <SRV_Service_URL>
```

Comment:
We also have the scalar-admin executable jar, right? We don't mention it?

Reply:
> We also have the scalar-admin executable jar, right? 

Yes

> We don't mention it?

Sorry, Now it is added